### DISCUSSION TITLE
Generals ID

### DISCUSSION THREAD

**Discussion by JohnRo79** [fedorovvl/tso_client#272] [Ideas]
hi there

can we have a tab showing every general's ID number?

https://prnt.sc/G-R93-VXrZlf

**Comment by Catavasiya** [comment]
you decided to spam the whole github? Specialists->Hide specialists->General
![image](https://github.com/fedorovvl/tso_client/assets/128522616/bd55706a-4601-4e0b-9e31-166a80bf9a81)
